client/ayon_core/plugins/load/export_otio.py

```diff
@@ -0,0 +1,280 @@
+            widget = QtWidgets.QPushButton(name)
+            input_layout.addWidget(
+                widget,
+                0,
+                count + 1,
+                alignment=QtCore.Qt.AlignCenter
+            )
+            widget.clicked.connect(self.toggle_all)
+
+        self._representation_widgets = defaultdict(list)
+        row = 1
+        items = representations_by_version_id.items()
+        for version_id, representations in items:
+            version_path = self._version_path_by_id[version_id]
+            input_layout.addWidget(QtWidgets.QLabel(version_path), row, 0)
+
+            representations_by_name = {x["name"]: x for x in representations}
+            group_box = QtWidgets.QGroupBox()
+            layout = QtWidgets.QHBoxLayout()
+            group_box.setLayout(layout)
+            for count, name in enumerate(all_representation_names):
+                if name in representations_by_name.keys():
+                    widget = QtWidgets.QRadioButton()
+                    self._representation_widgets[name].append(
+                        {
+                            "widget": widget,
+                            "representation": representations_by_name[name]
+                        }
+                    )
+                else:
+                    widget = QtWidgets.QLabel("x")
+
+                layout.addWidget(widget)
+
+            input_layout.addWidget(
+                group_box, row, 1, 1, len(all_representation_names)
+            )
+
+            row += 1
+
+        export_widget = QtWidgets.QWidget()
+        export_layout = QtWidgets.QVBoxLayout(export_widget)
+
+        widget = QtWidgets.QWidget()
+        layout = QtWidgets.QHBoxLayout(widget)
+        self.button_output_path = QtWidgets.QPushButton("Output Path:")
+        layout.addWidget(self.button_output_path)
+        self.lineedit_output_path = QtWidgets.QLineEdit()
+        layout.addWidget(self.lineedit_output_path)
+        export_layout.addWidget(widget)
+
+        self.checkbox_inspect_otio_view = QtWidgets.QCheckBox(
+            "Inspect with OTIO view"
+        )
+        export_layout.addWidget(self.checkbox_inspect_otio_view)
+
+        self.button_export = QtWidgets.QPushButton("Export")
+        export_layout.addWidget(self.button_export)
+
+        layout = QtWidgets.QVBoxLayout(self)
+        layout.addWidget(input_widget)
+        layout.addWidget(export_widget)
+
+        self.button_export.clicked.connect(self.export)
+        self.button_output_path.clicked.connect(self.set_output_path)
+
+    def toggle_all(self):
+        representation_name = self.sender().text()
+        for item in self._representation_widgets[representation_name]:
+            item["widget"].setChecked(True)
+
+    def set_output_path(self):
+        file_dialog = QtWidgets.QFileDialog()
+        file_dialog.setNameFilter("OTIO Files (*.otio)")
+        file_dialog.setFileMode(QtWidgets.QFileDialog.ExistingFile)
```

Comment:
```suggestion
        file_dialog.setFileMode(QtWidgets.QFileDialog.AnyFile)
```
This had fixed the issue with non existent file from comment.

Reply:
But to be honest it still feels bit confusing, since no file exists. I would rather recomend to split the attribute into two: Output Dir and File Name (without extension, since it is quite obvious). Where the File Name can be pre-created by some template in settings and perhaps manually changed if desired.

Reply:
Fixed